LICENSE.txt

```diff
@@ -0,0 +1,20 @@
+Copyright (C) Brian Nesbitt
```

Comment:
Was wondering about that myself but since I "borrowed" the LICENSE.txt from Chronis it might be related to Carbon. Will check/fix tomorrow. :thumbsup: